pkg/cri/server/base/cri_base.go

```diff
@@ -106,7 +108,9 @@ func initCRIBase(ic *plugin.InitContext) (interface{}, error) {
 		StateDir:           stateDir,
 	}
 
-	log.G(ctx).Infof("Start cri plugin with config %+v", c)
+	// Ignoring errors here; this should never fail.
+	cfg, _ := json.Marshal(c)
+	log.G(ctx).WithFields(log.Fields{"config": string(cfg), "id": pluginID, "type": plugins.InternalPlugin}).Info("starting cri plugin")
```

Comment:
I like using structured, but the id and plugin type actually don't match here. The `pluginID` is using the old id for compatibility. The relevant plugin is `io.containerd.internal.cri`, but I'm not so sure it is useful to show that.